pkg/kdump/kdump-view.jsx

```diff
@@ -26,7 +26,7 @@ import { OverlayTrigger, Tooltip } from "patternfly-react";
 import * as Select from "cockpit-components-select.jsx";
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 
-import "form-layout.less";
+import "form-layout.scss";
```

Comment:
This commit ("Convert all less to scss") is nice! At first sight this shouldn't fundamentally break anything and be fairly independent of the other workd. Could this become a separate PR that we land quickly, as it touches so many files and becomes stale quite fast?

Reply:
Sure - good point